bundle-3.4/pom.xml

```diff
@@ -0,0 +1,1344 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Want to make sure I've done this right - 

1. Ran the jars-pommefizer on spark-3.4.1-bin-hadoop3
2. Copied over some basic pom setup info from bundle-3.3 (higher level project template
3. Moved dependencies from pommefizer output into dependencyManagement 
4. Filled in dependencies whose groupIds were blank from pommefizer (copied from 3.3, maybe a bug in pommefizer?)
5. Copied build plugins from 3.3

Reply:
Seems alright, @wajda what do you think?